test/unit/client.test.js

```diff
@@ -32,15 +32,18 @@ test('should use uuid from auth key as space guid if apigw_token present', t =>
 test('should use environment parameters for options if not set explicitly.', t => {
   process.env['__OW_API_KEY'] = 'some_user:some_pass'
   process.env['__OW_API_HOST'] = 'mywhiskhost'
+  process.env['__OW_IGNORE_CERTS'] = true
```

Comment:
in node.js the environment variables are always strings, pls use string value instead if boolean